.github/workflows/dev-deploy.yaml

```
@@ -0,0 +1,213 @@
+        id: filter
+        with:
+          filters: |
+            token_refresh: 'cloud_functions/token_refresh/**'
+
+  cloud_function_deploy:
+    needs: cloud_function_detect_changes
+    runs-on: ubuntu-latest
+    environment: dev
+    strategy:
+      matrix:
+        function: [ 'token_refresh' ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)